### DISCUSSION TITLE
Closing a chia pool (https://azpool.org/u/dashboard)

### DISCUSSION THREAD

**Discussion by santom1977** [Chia-Network/chia-blockchain#8507] [Support]
Hi guys, I hope you are well.

I have a pool that is going to close (https://azpool.org/u/dashboard), but I have some plots mining there. Is it possible to change all the plots from this pool server to other pool server?

Thanks santom

**Comment by shaneo257** [comment]
yes you can
in the gui, you click on Pool, then Change Pool, or cli follow the below 
https://github.com/Chia-Network/chia-blockchain/wiki/CLI-Commands-Reference#plotnft

**Comment by BrandtH22** [comment]
@santom1977 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
